api/routes/user.js

```diff
@@ -359,10 +392,21 @@ router.get("/updates/:numUpdates/:currentIndex", checkToken, async function(
       }
     }
   );
+  let shouldNotif = false;
 
-  res
-    .status(SUCCESS)
-    .send(feedUpdates.slice(currentIndex, currentIndex + numUpdates));
+  for (let update of feedUpdates) {
+    if (update.date > user.lastCheckedNotifs) {
+      shouldNotif = true;
+    }
+  }
```

Comment:
nitpick but this section could be rewritten as:
```js
const shouldNotif = feedUpdates.some(update => update.date > user.lastCheckedNotifs);
```

Reply:
🤯